pkg/osquery/table/platform_tables_linux.go

```diff
@@ -13,5 +14,7 @@ func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, c
 	return []*table.Plugin{
 		gsettings.Settings(client, logger),
 		gsettings.Metadata(client, logger),
+		dataflattentable.TablePluginExec(client, logger,
+			"kolide_nmcli", dataflattentable.KeyValueType, []string{"/usr/bin/nmcli", "--mode=multiline", "--fields=all", "device", "wifi", "list"}, dataflattentable.WithSeperator(":")),
```

Comment:
Should this be `kolide_nmcli` or `kolide_nmcli_device` ?

Reply:
I think either `kolide_nmcli` or `kolide_nmcli_device_wifi` (maybe even `kolide_nmcli_device_wifi_list`), because `nmcli device` has quite a few other subcommands. 

Actually, how about `kolide_nmcli_wifi`? it leaves us open to add other wifi listing tables for linux **and** other `nmcli` tables, should we need them.

Reply:
I don't know the command well enough to feel strongly. Mostly I observed we should think about it. Renaming later is always an option